product_docs/docs/sql_assess/1/using.mdx

```diff
@@ -0,0 +1,260 @@
+
+- Location for output file &mdash; Provide the path where you want the EDB Query Extractor to place the SQL statements extracted from the `V$SQL` table. By default, the trace file generates from the location where SQLPlus was run from.
+
+- The name of the user whose queries are to be extracted &mdash; The script extracts the SQL statements run only by the provided Oracle user. By default, the script uses the current logged-in user.
+
+- The start timestamp of the queries &mdash; Provide the starting timestamp of the queries to extract in the format DD-MM-YYYY HH24:MI:SS. By default, the script uses `01-01-1970`.
+
+- The end timestamp of the queries &mdash; Provide the ending timestamp of the queries to extract in the format DD-MM-YYYY HH24:MI:SS. By default, the script uses the current time.
+
+The following is an example of the EDB Query Extractor input parameters:
+
+```
+SQL>@edb_queries_extractor.sql
+# -- EDB Query Extractor Version 1.0.0 for Oracle Database -- #
+# ----------------------------------------------------------- #
+Caution:
+Script USER must have CONNECT role and SELECT ANY DICTIONARY privilege.
+(To verify granted roles and privileges, run the following commands: SELECT granted_role FROM user_role_privs;  or  SELECT privilege FROM user_sys_privs;)
+Press RETURN to continue ...
+Location for output file (Default current location): /tmp/
+Enter the name of the user whose queries are to be extracted (Default current user): rs2_lin32
+Enter the start timestamp of the queries (DD-MM-YYYY HH24:MI:SS) (Default 01-01-1970): 
+Enter the end timestamp of the queries (DD-MM-YYYY HH24:MI:SS) (Default current time): 
+
+Writing queries to /tmp/RS2_LIN32_230518054450.out
+####################################################################################################
+## EDB Query Extractor Utility. Version: 1.0.0
+##
+## Source Database Version	: Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production
+## Queries Extracted for User	: rs2_lin32 (User Id: 109)
+## Time Duration for Extraction	: 01-01-1970 00:00:00 to 18-05-2023 05:44:50
+## Extracted On			: 18-05-2023 05:44:50
+####################################################################################################
+Disconnected from Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production
+Version 19.3.0.0.0
+
+```
+
+## Performing a SQL assessment
+
+After generating the trace files using either the Oracle trace utility or the EDB Query Extractor, the user can run the `assessSQL.jar` file using the `runSQLAssess.sh` script to generate a report. The report helps the user to identify the compatibility of the application for porting the application to EDB Postgres Advanced Server. The tool also generates SQL output files. The queries in these output files can then be modified to be compatible with EDB Postgres Advanced Server syntax. 
+
+!!! Note 
```

Comment:
The following note seems misplaced.  It might be better after the **Command options** section